### ISSUE TITLE
coco格式数据集训练准确度很低

### ISSUE THREAD

**Issue by JasenWangLab** [Megvii-BaseDetection/YOLOX#259]
你好，感谢你们工作的开源，我把自己的数据转换成coco格式后，在大约20epoch的时候，训练100个epoch准确度还是很低，total_loss基本就不变了，请问下会是什么原因导致的呢
![image](https://user-images.githubusercontent.com/76416039/127584631-15d0fb9f-1d23-475c-be02-fa90b230da76.png)
这个是我的训练命令：python tools/train.py -n yolox_tiny -f /YOLOX-main/exps/example/yolox_voc/yolox_my.py -d 2 -b 16 --fp16 -o

**Comment by ruinmessi** [comment]
Plz load the pretrained COCO weights using `-c /path/to/the/weights`

**Comment by JasenWangLab** [comment]
> Plz load the pretrained COCO weights using `-c /path/to/the/weights`

OK, thank you very much. I'll try this method.But if I want to train from scratch with my own dataset, is there any solution to this problem

**Comment by BowieHsu** [comment]
I trained yolox tiny on my datasets and meet the same problem, maybe caused by init setting about bn: "m.eps = 1e-3 m.momentum = 0.03", I also try to init weights with coco pretrained model , waiting for  eval results now.

**Comment by Joker316701882** [comment]
@BowieHsu @JasenWangLab Training from scratch is not recommended, unless your training data is as large as COCO dataset.

**Comment by TUDelftHao** [comment]
> 你好，感谢你们工作的开源，我把自己的数据转换成coco格式后，在大约20epoch的时候，训练100个epoch准确度还是很低，total_loss基本就不变了，请问下会是什么原因导致的呢
> ![image](https://user-images.githubusercontent.com/76416039/127584631-15d0fb9f-1d23-475c-be02-fa90b230da76.png)
> 这个是我的训练命令：python tools/train.py -n yolox_tiny -f /YOLOX-main/exps/example/yolox_voc/yolox_my.py -d 2 -b 16 --fp16 -o

I am also suffering from this trouble. Even when I load the pretrained model of yolox-s, the loss stabilizes around 10.0 and the eval map is approaching 0, which is even worse than the beginning (about 0.45). @Joker316701882 @ruinmessi Do you know what causes this phenomenon ? @JasenWangLab Did you manage to solve this problem?

**Comment by mikel-brostrom** [comment]
Same problem here as @TUDelftHao 

![Screenshot from 2021-08-10 10-39-46](https://user-images.githubusercontent.com/18719680/128835961-1d0575cc-99e7-4f66-b36c-505ce4f5b45f.png)

**Comment by wang1233211** [comment]
兄弟，有好的解决方案吗
